### PR TITLE
add classNamed: in BaselineOf

### PR DESCRIPTION
**Pull request by Ducasse**
use classNamed: where needed.
Should do Spec2 and NewTools baselines too.

### CODE REVIEW

**Comment by Ducasse** [comment]
I will add first classNamed: alone because there may be some dependencies.

**Comment by Ducasse** [comment]
Let us now that the baselineOf new behavior has been added in another PR if this one is still not building.

**Comment by jecisc** [comment]
There seems to be a problem with this change:

Instance of SystemEnvironment did not understand #specialObjectsArray
SystemEnvironment(Object)>>doesNotUnderstand: #specialObjectsArray
BaselineOfDisplay>>installBitmap
BaselineOfDisplay>>postload:package:

File: src/BaselineOfDisplay/BaselineOfDisplay.class.st [hunk 1]
```diff
@@ -47,10 +47,10 @@ BaselineOfDisplay >> installBitmap [
 
 	| array |
 	
-	array := Smalltalk specialObjectsArray copy.
+	array := self class environment specialObjectsArray copy.
```
Comment (jecisc) [inline src/BaselineOfDisplay/BaselineOfDisplay.class.st:50]: It's not the environment that knows the special object array but the image

File: src/BaselineOfDisplay/BaselineOfDisplay.class.st [hunk 2]
```diff
@@ -47,10 +47,10 @@ BaselineOfDisplay >> installBitmap [
...
 	array at: 5 put: Bitmap.
 
-	Smalltalk specialObjectsArray becomeForward: array.
+	self class environment specialObjectsArray becomeForward: array.
```
Comment (jecisc) [inline src/BaselineOfDisplay/BaselineOfDisplay.class.st:53]: Same

File: src/BaselineOfIDE/BaselineOfIDE.class.st [hunk 3]
```diff
@@ -196,18 +190,21 @@ BaselineOfIDE >> loadExtraTools [
 { #category : 'actions' }
 BaselineOfIDE >> loadIceberg [
 
+	| os |
+	os := self class environment os.
```
Comment (jecisc) [inline src/BaselineOfIDE/BaselineOfIDE.class.st:194]: We should use `self os` since it's now in BaselineOf

**Review by jecisc — CHANGES_REQUESTED** [review]

File: src/BaselineOfDisplay/BaselineOfDisplay.class.st (hunk 1 above)
Comment (Ducasse) [reply src/BaselineOfDisplay/BaselineOfDisplay.class.st:50]: Yes my mistake I will redo this PR.

File: src/BaselineOfIDE/BaselineOfIDE.class.st (hunk 3 above)
Comment (Ducasse) [reply src/BaselineOfIDE/BaselineOfIDE.class.st:194]: Yes!

**Pull request closed without merging**